src/data/services/EcommerceApiService.js

```diff
@@ -86,6 +86,18 @@ class EcommerceApiService {
     }
     return EcommerceApiService.apiClient().post(url, options);
   }
+
+  static fetchEnterpriseOffers(options = {
+    status: 'Open',
+  }) {
+    const { enterpriseId } = store.getState().portalConfiguration;
+    let url = `${EcommerceApiService.ecommerceBaseUrl}/api/v2/enterprise/${enterpriseId}/enterprise-admin-offers/`;
+    if (options) {
```

Comment:
[curious] If `options` has a default function argument to fallback on, will `options` ever be falsy here?

Reply:
Agreed, sounds good :)